### PR TITLE
Detect pseudo/v4

### PR DESCRIPTION
**Pull request by victorjulien**
#11088 rebased with some cleanups

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1904

https://redmine.openinfosecfoundation.org/issues/7002

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information:
 

ERROR: QA failed on SURI_TLPW2_autofp_suri_time.
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPW2_autofp_stats_chk | | |
| .uptime | 137 | 142 | 103.65% | 

Pipeline 21012

File: src/detect-stream_size.c [hunk 1]
```diff
@@ -214,9 +211,9 @@ static bool PrefilterPacketStreamSizeCompare(PrefilterPacketHeaderValue v, void
 
 static int PrefilterSetupStreamSize(DetectEngineCtx *de_ctx, SigGroupHead *sgh)
 {
-    return PrefilterSetupPacketHeader(de_ctx, sgh, DETECT_STREAM_SIZE,
-            SIG_MASK_REQUIRE_FLOW | SIG_MASK_REQUIRE_REAL_PKT, PrefilterPacketStreamSizeSet,
-            PrefilterPacketStreamSizeCompare, PrefilterPacketStreamsizeMatch);
+    return PrefilterSetupPacketHeader(de_ctx, sgh, DETECT_STREAM_SIZE, SIG_MASK_REQUIRE_FLOW,
```
Comment (catenacyber) [inline src/detect-stream_size.c:214]: Is this a new feature ? Is there a test ?

**Review by catenacyber — APPROVED** [review]
Thanks for the nice work :-)

- CI : 🟢 
- Code : good for me
- Commits segmentation : nice
- Commit messages : good :-)
- Git ID set : looks fine for me
- CLA : you already contributed :-p
- Doc update : not needed
- Redmine ticket : ok
- Rustfmt : no rust
- Tests : SV 🟠  on 6 otherwise ok
- Dependencies added: none

File: src/detect-stream_size.c (hunk 1 above)
Comment (victorjulien) [reply src/detect-stream_size.c:214]: I see it as an internal optimization.

**Comment by victorjulien** [comment]
Replaced by #11318

**Pull request closed without merging**